src/main/kotlin/com/exactpro/th2/kafka/client/KafkaConnection.kt

```diff
@@ -224,6 +234,28 @@ abstract class KafkaConnection<MESSAGE, MESSAGE_BUILDER>(
 
     override fun close() = producer.close()
 
+    private fun createTopicOffsets(topic: String) =
+        TopicOffsets { partition, prevOffset, newOffset ->
+            eventSender.onEvent(
+                name = "Gap detected for topic '$topic' in partition '$partition' between $prevOffset and $newOffset",
```

Comment:
Maybe add information about subscription group

Reply:
@Nikita-Smirnov-Exactpro Do you mean `consumer group`?